src/NuGet.Clients/PackageManagement.VisualStudio/PackageManagement.VisualStudio.csproj

```diff
@@ -49,16 +49,36 @@
     <Reference Include="Microsoft.VisualStudio.ComponentModelHost">
       <HintPath>$(ProgramFiles)\Microsoft Visual Studio $(VisualStudioVersion)\Common7\IDE\PublicAssemblies\Microsoft.VisualStudio.ComponentModelHost.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.VisualStudio.ProjectSystem.V14Only">
-      <HintPath>..\..\..\lib\VS14\Microsoft.VisualStudio.ProjectSystem.V14Only.dll</HintPath>
-    </Reference>
-    <Reference Include="Microsoft.VisualStudio.ProjectSystem.Interop">
-      <HintPath>..\..\..\lib\Microsoft.VisualStudio.ProjectSystem.Interop.dll</HintPath>
-      <EmbedInteropTypes>True</EmbedInteropTypes>
-    </Reference>
+  </ItemGroup>
+  <Choose>
```

Comment:
Consuming project system libraries here makes this assembly a prime candidate for taking a dependency on the facade assembly in future. For now these libraries need to stay as msbuild dependencies until we support embedding interop types (per main message on this PR).